src/lib/Microsoft.Health.Extensions.Fhir.R4/Telemetry/Exceptions/FhirServiceExceptionProcessor.cs

```diff
@@ -41,8 +40,8 @@ public static (Exception customException, string errorName) CustomizeException(E
 
             switch (exception)
             {
-                case FhirOperationException _:
-                    var status = ((FhirOperationException)exception).Status;
+                case HttpRequestException _:
```

Comment:
TIL, thanks!

Reply:
Thanks for calling this out! Please refer to my [comment](https://github.com/microsoft/iomt-fhir/pull/167#discussion_r803212593) below providing context on the latest changes for this class.

Reply:
Isn't clear to me how comment address the cast when it isn't needed.  

Instead of:
case FhirException _:
                    var status = ((FhirException)exception).StatusCode;

you can just do
case FhirException fhirEx:
                    var status = fhirEx.StatusCode;